src/pages/home/ReportScreenContext.ts

```diff
@@ -0,0 +1,17 @@
+import {RefObject, createContext} from 'react';
+import {FlatList, GestureResponderEvent} from 'react-native';
+
+type ReactionListRefType = {
```

Comment:
Do not suffix types with `Type`. From guidelines:
```ts
  - Do not postfix type aliases with `Type`.

    ```ts
    // BAD
    type PersonType = ...;

    // GOOD
    type Person = ...;
    ```
```

Reply:
Right, updated!